### ISSUE TITLE
Periodic timer instead of number of requests

### ISSUE THREAD

**Issue by radical-equanimity** [kzk/unicorn-worker-killer#68]
Running Gitlab on a memory-constrained RPI2, where unicorn-worker-killer is used to restart workers after every X requests. Problem is that requests are infrequent and memory leaks accumulate while worker is running (even without any requests). Would be great to have option to set a time interval for the memory check instead of only doing it every X requests.

**Comment by lacostenycoder** [comment]
I tend to agree that number of requests alone seems like an arbitrary method of deciding when to restart.

**Comment by marcosArruda** [comment]
+1
